en/debug/sensor_uorb_topic_debugging.md

```diff
@@ -0,0 +1,17 @@
+# Sensor/Topic Debugging using the Listener Command
+
+The uORB is an asynchronous `publish()` / `subscribe()` messaging API used for
+inter-thread/inter-process communication. The `listener` command can be used from the *QGroundControl MAVLink Console* to inspect topic (message) values, including the current values published by sensors.
```

Comment:
I'd probably start more general, that the `listener` command is available on the shell (you mention it at the bottom), and then go on to the QGC console (which is actually much more powerful as well, as you can monitor system load, start/stop tasks, etc. directly from QGC).